### PR TITLE
db changes for va caution flags

### PR DESCRIPTION
**Pull request by hughes-dustin**
## Description
https://app.zenhub.com/workspaces/vft-59c95ae5fda7577a9b3184f8/issues/department-of-veterans-affairs/va.gov-team/10883

Creates the `va_caution_flags` table.

## Definition of done
- [ ] Events are logged appropriately
- [ ] Documentation has been updated, if applicable
- [ ] A link has been provided to the originating GitHub issue (or connected to it via ZenHub)
- [ ] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs

### CODE REVIEW

File: db/migrate/20200718131012_create_va_caution_flags.rb [hunk 1]
```diff
@@ -0,0 +1,17 @@
+
+class CreateVaCautionFlags < ActiveRecord::Migration[4.2]
+    def change
+      create_table :va_caution_flags do |t|
+        t.string :institution_id, null: false
+        t.string :institution_name
+        t.integer :school_system_code
```
Comment (zurbergram) [inline db/migrate/20200718131012_create_va_caution_flags.rb:7]: i'm not seeing this column in the story or sample file

File: db/migrate/20200718131012_create_va_caution_flags.rb [hunk 2]
```diff
@@ -0,0 +1,17 @@
+
+class CreateVaCautionFlags < ActiveRecord::Migration[4.2]
+    def change
+      create_table :va_caution_flags do |t|
+        t.string :institution_id, null: false
```
Comment (zurbergram) [inline db/migrate/20200718131012_create_va_caution_flags.rb:5]: should this be called `facility_code` so as to not confuse it with be a foreign key column that relates to the `institutions` table?

Comment (hughes-dustin) [reply db/migrate/20200718131012_create_va_caution_flags.rb:5]: I changed it.

**Review by shawkey-daniel — APPROVED** [review]
LGTM

**Review by omgitsbillryan — APPROVED** [review]
LGTM, but make sure you're careful not to use the `VaCautionFlag` model anywhere in code until the migration code is deployed and exectured.

**Pull request merged**